src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -410,17 +410,13 @@ public DataFile getInitialDataFile(){
         return null;
     }
     
-    public String getSwiftContainerName(){
-
-        String swiftContainerName;
+    public SwiftAccessIO getSwiftObject() {
```

Comment:
We should figure out if this gets merged before or after 3919, because either way, we need to change this to use SwiftAccessIO parametrized by the correct type.